kevm

```diff
@@ -56,9 +56,12 @@ run_prove() {
 
     def_module="$1" ; shift
 
+    repl_cmd="${repl_cmd:-$k_release_dir/bin/kore-repl}"
+    repl_script="${repl_script:-$kevm_dir/kast.kscript}"
```

Comment:
```suggestion
    repl_cmd="${REPL_CMD:-$k_release_dir/bin/kore-repl}"
    repl_script="${REPL_SCRIPT:-$kevm_dir/kast.kscript}"
```

Also add `repl_cmd` and `repl_script` to the `local ...` declaration above.

Reply:
sounds good. done.